spot_rl_experiments/spot_rl/envs/skill_manager.py

```diff
@@ -145,16 +153,16 @@ def pick(self, pick_target: str = None) -> Tuple[bool, str]:
         print(f"Picking {pick_target}")
 
         result = None
-        try:
-            result = self.gaze_controller.execute([pick_target])
-        except Exception:
-            return False, "Error encountered while picking"
+        #try: try & catch was nerfing the code failure
```

Comment:
We still need to do `try-catch` to output the error message. The reason is that in the coming weeks, we also want to integrate Large Language Models (LLMs) into the system, and the LLMs will need text feedback from the system.